test/test_transformers.py

```diff
@@ -1871,11 +1872,11 @@ def test_scaled_dot_product_fused_attention_vs_math_cpu(
     @onlyCPU
     @parametrize("fused_kernel", [SDPBackend.FLASH_ATTENTION])
     @parametrize("dtype", [torch.float64, torch.float32, torch.bfloat16, torch.float16])
-    @parametrize("batch_size", [2, 12])
-    @parametrize("q_seq_len", [267, 1030])
-    @parametrize("kv_seq_len", [514, 1179])
-    @parametrize("n_head", [1, 3])
-    @parametrize("head_dim", [8, 16])
+    @parametrize("batch_size", [2])
+    @parametrize("q_seq_len", [267])
+    @parametrize("kv_seq_len", [514])
+    @parametrize("n_head", [3])
```

Comment:
Removing the tests here I assume for speed? Since we are adding more additional tests for every parametrized input?

Reply:
Exactly! The test with mask becomes very large because of different mask shapes and types. In addition, different input shapes are already included in the non-mask version test.